ansible/roles/learning-service/tasks/main.yml

```diff
@@ -10,13 +18,18 @@
 - name: Delete existing configuration files
   become: yes
   become_user: "{{learner_user}}"
-  file: name=/data/properties/learning state=absent
+  file: name=/data/properties/application.conf state=absent
 
 - name: Copy application.conf file
   become: yes
   become_user: "{{learner_user}}"
   template: src=application.conf.j2 dest=/data/properties/application.conf mode=755 owner={{ learner_user }} group={{ learner_group }}
 
+- name: Delete existing artifact file from tomcat webapps folder
```

Comment:
We should delete the folder also.

Reply:
task for deleting learning-service folder was alredy there, we are deleting folder before we start tomcat.